src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1396,6 +1396,31 @@ class CreateIndexStatementSegment(BaseSegment):
     )
 
 
+@postgres_dialect.segment()
+class AnalyzeStatementSegment(BaseSegment):
+    """Analyze Statement Segment.
+
+    As specified in https://www.postgresql.org/docs/13/sql-analyze.html
+    """
+
+    type = "analyze_statement"
+
+    _option = Sequence(
+        OneOf("VERBOSE", "SKIP_LOCKED"), Ref("BooleanLiteralGrammar", optional=True)
+    )
+
+    _tables_and_columns = Sequence(
+        Ref("TableReferenceSegment"),
+        Bracketed(Delimited(Ref("ColumnReferenceSegment")), optional=True),
+    )
+
+    match_grammar = Sequence(
+        OneOf("ANALYZE", "ANALYSE"),
+        OneOf(Bracketed(Delimited(_option)), "VERBOSE", optional=True),
```

Comment:
Did you see this bit:
> When the option list is surrounded by parentheses, the options can be written in any order. The parenthesized syntax was added in PostgreSQL 11; the unparenthesized syntax is deprecated.

Should we support the unparenthesized in case any on else is still using it? Or is it deprecated enough that we don't need to support it?

```suggestion
        OneOf(Bracketed(Delimited(_option)), Delimited(_option), "VERBOSE", optional=True),
```

Reply:
The unparenthesized syntax it's referring to is just VERBOSE. Parenthesized was introduced in pg11, and SKIP_LOCKED as an option was introduced in pg12. So there is no postgres version where unparenthesized SKIP_LOCKED or VERBOSE boolean would run

Reply:
Ah gotcha. Resolving.